apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -161,8 +161,7 @@ export default function Success(props: PageProps) {
   const [calculatedDuration, setCalculatedDuration] = useState<number | undefined>(undefined);
   const [comment, setComment] = useState("");
   const parsedRating = rating ? parseInt(rating, 10) : 3;
-  const currentUserEmail =
-    searchParams?.get("cancelledBy") ?? searchParams?.get("email") ?? session?.user?.email ?? undefined;
+  const currentUserEmail = searchParams?.get("cancelledBy") ?? session?.user?.email ?? undefined;
```

Comment:
Will this also work for `rescheduledBy`?

Reply:
Yes if we use the `currentUserEmail` and append it the url when redirecting to reschedule page

https://github.com/calcom/cal.com/blob/f11086e9f8fb4eae41431cb6b5ce6dd356aed0fb/apps/web/modules/bookings/views/bookings-single-view.tsx#L703-L709